xbmc/events/EventLog.cpp

```diff
@@ -199,7 +191,8 @@ void CEventLog::Remove(const std::string& eventPtrIdentifier)
 
   EventPtr eventPtr = itEvent->second;
   m_eventsMap.erase(itEvent);
-  std::remove_if(m_events.begin(), m_events.end(), [eventPtr](const EventPtr& otherEvent) { return eventPtr == otherEvent; });
+  auto it = std::remove_if(m_events.begin(), m_events.end(), [eventPtr](const EventPtr& otherEvent) { return eventPtr == otherEvent; });
```

Comment:
Shouldn't just `std::remove` be sufficient? The default is to remove by equality